packages/web-components/src/components/ic-footer-link/ic-footer-link.css

```diff
@@ -90,3 +82,18 @@
 :host(.breadcrumb-link.current-page) .ic-link:visited {
   color: var(--ic-color-primary-text);
 }
+
+:host(.footer-link-light) ::slotted(svg) {
+  fill: var(--ic-architectural-white);
+}
+
+:host(.footer-link-dark) ::slotted(svg) {
+  fill: var(--ic-architectural-black);
```

Comment:
Rather than using those architectural white\black tokens, does it work if you just use --ic-theme-text? As that should get calculated to be the appropriate color

Reply:
Good suggestion! Tried it out locally and it seems to work :)